lib/components/dashboard/degree-info/DistributionBarsJSX.tsx

```diff
@@ -237,13 +237,19 @@ const DistributionBarsJSX: FC<{ major: Major }> = ({ major }) => {
     req: requirements,
     reqGroup: [string, requirements[]],
   ) => {
-    if (req.pathing) {
-      let [requirement, ...focus_areas] = reqGroup[1];
-      for (let focus_area of focus_areas) {
-        if (focus_area.fulfilled_credits >= focus_area.required_credits) {
-          reqGroup[1] = [requirement, focus_area];
+    if (req.pathing !== undefined) {
```

Comment:
This change should be unneeded. req.pathing checks if req.pathing !== undefined.